.gitignore

```diff
@@ -1,7 +1,11 @@
-### Composer ###
+# Composer
 composer.phar
 /vendor/
+
+# PHPStorm
 /.idea/
-/Tests/cache/
+
+# Unit Tests
+/tests/cache/
```

Comment:
Weirdly enough, `.phpunit.result.cache` can be generated, too. Can you add it here, too, please?